packages/apollo-codegen-swift/src/codeGeneration.ts

```diff
@@ -172,7 +175,7 @@ export class SwiftAPIGenerator extends SwiftGenerator<CompilerContext> {
       options: { namespace },
       fragments
     } = this.context;
-    const isRedundant = !!namespace;
+    const isRedundant = !!namespace && !outputIndividualFiles;
```

Comment:
Isn't the codegen written with extensions when `outputIndividualFiles` is true? In that case this should be
```swift
const isRedundant = !!namespace && outputIndividualFiles;
```
right?

Reply:
You're correct that it's written with extensions when `outputIndividualFiles` is true, but `isRedundant` should only ever be true if we're **not** outputting individual files.

If `isRedundant` is true, codegen omits the `public`, which is the behavior filed as a bug.

Reply:
After figuring out the thing I fucked up, yep, you're right!